client/app/index.jsx

```diff
@@ -1,4 +1,5 @@
 import './lib/polyfills'
+import 'babel-polyfill'
```

Comment:
We should probably remove lib/polyfills as the polyfills are auto-loaded via through Babel now?